azure-pipelines/codesync-msft-submodules.yml

```diff
@@ -35,6 +35,8 @@ steps:
     displayName: Setup env
   - bash: |
       set -ex
+      # 202506 only code sync these 4 repos.
+      branches_202506="sonic-utilities sonic-host-services sonic-platform-common sonic-platform-daemons"
```

Comment:
hi @liushilongbuaa does this mean, we are able to sync these 4 repos from 202505? @vvolam for viz.